.github/workflows/ci.yml

```diff
@@ -104,6 +108,12 @@ jobs:
       - name: "Install dependencies"
         run: python -m pip install --upgrade pip setuptools nox
 
+      - name: "Install Chrome"
+        uses: browser-actions/setup-chrome@11cef13cde73820422f9263a707fb8029808e191
```

Comment:
Can you please also add the version as a comment, as in the "Setup Python" and "Checkout repository" steps above? This will also be updated by dependabot, and allows humans to know what version we're using.

Reply:
Think I did this right now